src/components/Checkbox/Checkbox.js

```diff
@@ -133,6 +127,7 @@ const Checkbox = ({ children, id: customId, className, ...props }) => {
       <CheckboxInput {...props} id={id} type="checkbox" />
       <CheckboxLabel {...props} htmlFor={id}>
         {children}
+        <CheckedIcon />
```

Comment:
The SVG should be hidden from screen readers since it reading it out might be confusing — the `checked` state of the `<input />` communicates the state enough.

```suggestion
        <CheckedIcon aria-hidden="true" />
```